library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -61,6 +61,15 @@ protected function assembleIconImage(BaseHtmlElement $iconImage)
         if (isset($this->item->icon_image->icon_image)) {
             $src = $this->item->icon_image->icon_image;
 
+            if (strpos($src, '.') === false) {
+                $iconImage->add(HtmlElement::create('i', [
+                    'class'     => 'fas fa-' . $src,
+                    'alt'       => $this->item->icon_image_alt
+                ]));
```

Comment:
There's the `Icon` class, can't you use that? Also, `icon_image_alt` is of no use here, since the `alt` property is exclusive to `img` elements. For icons it's of no use, because it's mainly used to show something in case the `src` attribute of an image cannot be loaded. With icons there is no *loading* involved here.